src/main/java/com/disraely/entities/BaseEntity.java

```diff
@@ -0,0 +1,36 @@
+package com.disraely.entities;
+
+import java.io.Serializable;
+import java.util.Date;
+
+public abstract class BaseEntity {
+    private Long id;
+    private Date createDate;
```

Comment:
Take more advantage of new java versions by using `Instant` instead of `Date`

Reply:
Resolved 🙌